Sources/BraintreeCore/BTHTTPErrorCode.swift

```diff
@@ -1,36 +0,0 @@
-import Foundation
-
-@objc public enum BTHTTPErrorCode: Int {
```

Comment:
This was exposed publicly for use in Obj-C modules. We can add a changelog for removing this enum though if we feel it's ncessary, but probably should have been a `nodoc`.